tests/CMakeLists.txt

```diff
@@ -527,7 +527,7 @@ set(PYBIND11_TEST_PREFIX_COMMAND
     CACHE STRING "Put this before pytest, use for checkers and such")
 
 set(PYBIND11_PYTEST_ARGS
-    ""
+    "-v"
```

Comment:
Oh, interesting. I wasn't aware of this option.

This will change the behavior for people working interactively. Maybe it's better to systematically change the commands in .github/workflows/ci.yml instead? That'll only affect the GitHub Actions logs.

@henryiii WDYT?

For Henry:

We saw this error (:snake: graalpy-24.1 • ubuntu-20.04 • x64):

* https://github.com/pybind/pybind11/actions/runs/11787901190/job/32834040479?pr=5440

It was "fixed" simply by rerunning:

* https://github.com/pybind/pybind11/actions/runs/11787901190/job/32834781267?pr=5440

The logs from the first run don't have enough information for us to guess what might have gone wrong.

Reply:
Done